tests/foreman/api/test_subscription.py

```diff
@@ -355,3 +355,59 @@ def test_positive_candlepin_events_processed_by_stomp(rhel7_contenthost, functio
     response = entities.Ping().search_json()['services']['candlepin_events']
     assert response['status'] == 'ok'
     assert '0 Failed' in response['message']
+
+
+def test_positive_expired_SCA_cert_handling(module_org, rhel7_contenthost):
+    """Verify that a content host with an expired SCA cert can
+        re-register successfully
+
+    :id: 27bca6b8-dd9c-4977-81d2-319588ee59b3
+
+    :steps:
+
+        1. Import an SCA-enabled manifest
+        2. Register a content host to the Default Organization View using an activation key
+        3. Unregister the content host
+        4. Enable and synchronize a repository
+        5. Re-register the host using the same activation key as in step 3 above
+
+    :expectedresults: the host is re-registered successfully and its SCA entitlement
+                      certificate is refreshed
+
+    :Assignee: dsynk
+
+    :BZ: 1949353
```

Comment:
This BZ has customer cases attached, please set `CustomerScenario: true` token too

Reply:
Done.